cmd/timescale-reader/main.go

```diff
@@ -67,7 +67,7 @@ func main() {
 
 	repo := newService(db, logger)
 
-	auth, authHandler, err := authClient.Setup(envPrefix, svcName, cfg.JaegerURL)
+	auth, authHandler, err := authClient.Setup(envPrefix, cfg.JaegerURL, svcName)
```

Comment:
this addressed here https://github.com/mainflux/mainflux/pull/1834/files#diff-333e96fd1321194ba79c77c22f58e2746d699543d516fa8697006dcd39c57902 , could result in duplicates/conflicts

Reply:
Reverted.